porcupine/plugins/python_tools.py

```diff
@@ -63,6 +63,21 @@ def format_code_in_textwidget(tool: str, tab: tabs.FileTab) -> None:
             tab.textwidget.replace("1.0", "end - 1 char", after)
 
 
+black_format_tab = actions.register_filetab_action(
```

Comment:
These should be in `setup()`, so that they will not run when the plugin is disabled, even if someone decides to import it for some reason.

Reply:
So I'm not sure I agree with this. This kind of goes back to the whole "should it be possible to access actions from plugins directly" thing.

That said, I'll go ahead and move them into there now for the initial implementation and we can debate the merits of having them available outside actions module another time ;)

Reply:
1edd389